test/OpenTelemetry.Tests/Instrumentation/ActivityInstrumentationHelperTest.cs

```diff
@@ -22,18 +22,4 @@ public void SetActivitySource(string name, string version)
         Assert.Equal(activitySource.Version, activity.Source.Version);
```

Comment:
@Kielek - I think we can remove this file completely. `ActivityInstrumentationHelper.SetActivitySourceProperty` is only used in `TracerProviderSdkTest.cs`. We could move the code from `ActivityInstrumentationHelper` to `TracerProviderSdkTest.cs` and remove `ActivityInstrumentationHelper` as well.

Reply:
Fixed in b30490f4033531d6b6190cf1ba6c8368844d853c